### ISSUE TITLE
I still cant connect to the PC.

### ISSUE THREAD

**Issue by tungstenboy** [wileyyugioh/Osu-Touchpad#3]
I have followed all the instructions but still couldnt connect to the PC through both phone and iPad. Please help. Ive also checked firewall settings.

**Comment by wileyyugioh** [comment]
Can you double check that:

1. The iPad and the computer are connected to the same wifi network
2. You're able to connect via localhost and ip address to the app on the pc

**Comment by tungstenboy** [comment]
Theyre connected to the same network, i dont get this: You're able to connect via localhost and ip address to the app on the pc

**Comment by wileyyugioh** [comment]
When the program is running it gives an ip address and port number. When you type in this ip address into a web browser, does a web page show up?

**Comment by tungstenboy** [comment]
Yes. I went there, it doesnt load. After sometime it says cant find server or something like that

**Comment by wileyyugioh** [comment]
That implies there is either a firewall problem or your router is messing with the connection. Most likely it's a firewall problem.

**Comment by tungstenboy** [comment]
I 'Allowed access' to everything.

**Comment by wileyyugioh** [comment]
Go to localhost:3000 in your browser. Does a webpage appear? If so, then the program is running correctly. If not, then there's probably a problem with your ports on your computer.

Go to IPADDRESS:3000 in your browser. Does a webpage appear? If so, then the program is going through your firewall. If not, then your firewall is blocking the application.

I'm sorry, but this is all the help I can give. I don't know your particular computer setup.

**Comment by nyoemii** [comment]
Hey! Same problem here. I can connect on my PC to localhost:3000 and IPADRESS:3000 but on my Tablet it doesn't load, it says, that IPADRESS:3000 and localhost:3000 aren't available.

**Comment by meowchanicyuumi** [comment]
If you're on Mac and the IP is set to http://null, just reload the window. That should change the IP to: "[your computer's ip]:3000". Otherwise, if you're on Windows, I can't really help you (Sorry!).

**Comment by stylinnile** [comment]
I know this is old but I have the same issue. My ip starts with a 25.5- whereas on my laptop where it works it starts with the normal 192.- is there some network setting on my computer that is causing problems here?

**Comment by jackthewolf1** [comment]
can it be fixed for acer
